Src/WindowsServer/WindowsServer.Shared/Implementation/MetricManager.cs

```diff
@@ -409,7 +409,14 @@ private static MetricTelemetry CreateAggregatedMetricTelemetry(Metric metric, Si
             {
                 foreach (KeyValuePair<string, string> property in metric.Dimensions)
                 {
-                    telemetry.Properties.Add(property);
+                    if (string.Compare(property.Key, "operationName") == 0)
```

Comment:
thinking about it - maybe use `ai.operation,name` as we do here: https://github.com/Microsoft/ApplicationInsights-dotnet/blob/de488729c28b721a758e0024ea2d4f79909ff797/src/Core/Managed/Shared/Extensibility/Implementation/External/ContextTagKeys_types.cs#L174